src/Interface/InterfacesList.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
```

Comment:
is it really your task?

Reply:
We discovered a week ago that we have no separate task for this and this part is actually mentioned in this very ticket's description. Andrei decided to handle this all under one PR